.github/workflows/linters.yaml

```diff
@@ -21,5 +21,5 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Run ansible-lint
-        uses: ansible/ansible-lint@v6.21.0
\ No newline at end of file
+      - name: run-ansible-lint
```

Comment:
Why the change of step name?

Reply:
Replaced it with new task from https://github.com/marketplace/actions/run-ansible-lint

Reply:
<img width="768" alt="image" src="https://github.com/user-attachments/assets/99dc3e38-8915-4d2e-9fd4-afd29cdc8b12">